src/lib.rs

```diff
@@ -10,21 +10,32 @@ use glib::translate::{ToGlibPtr};
 pub struct AppIndicator {
     air: *mut AppIndicatorRaw
 }
-
+pub enum AppIndicatorCategory{
```

Comment:
While I doubt they're ever gonna change, shouldn't these map to the consts in libappindicator_sys, i.e. ApplicationStatus = AppIndicatorCategory_APP_INDICATOR_CATEGORY_APPLICATION_STATUS?

Reply:
I had the same question too. I think those C library dev-ers usually don't change those kinds of constants.

Reply:
Yeah, good enough for now.